src/schematools/factories.py

```diff
@@ -61,6 +61,10 @@ def tables_factory(
     metadata = metadata or MetaData()
     dataset_tables = dataset.get_tables(include_nested=True, include_through=True)
 
+    # For compatibility and consistency, allow snake-cased table ID's
+    db_table_names = _snake_keys(db_table_names)
+    db_schema_names = _snake_keys(db_schema_names)
```

Comment:
If we do

```suggestion
    db_table_names = _snake_keys(db_table_names or {})
    db_schema_names = _snake_keys(db_schema_names or {})
```

we don't need the `if` inside `_snake_keys`.

Reply:
done!